daemon/main.c

```diff
@@ -731,6 +731,11 @@ static int load_netdata_conf(char *filename, char overwrite_used) {
     return ret;
 }
 
+// coverity[ +tainted_string_sanitize_content : arg-0 ]
+inline void coverity_remove_taint(char *s)
+{
```

Comment:
Please add inside this function `(void)s;` to remove the warning

```
daemon/main.c: In function 'coverity_remove_taint':
daemon/main.c:735:41: warning: unused parameter 's' [-Wunused-parameter]
  735 | inline void coverity_remove_taint(char *s)
```

when Netdata is compiled with the following command line:

```
CFLAGS="-O1 -ggdb -Wall -Wextra -Wformat-signedness -fstack-protector-all -DNETDATA_INTERNAL_CHECKS=1 -D_FORTIFY_SOURCE=2 -DNETDATA_VERIFY_LOCKS=1" ./netdata-installer.sh --disable-go --disable-lto --dont-wait --dont-start-it
```

Reply:
Done.